scylla/src/transport/session.rs

```diff
@@ -31,9 +31,7 @@ use crate::transport::connection::{
 };
 use crate::transport::connection_pool::PoolConfig;
 use crate::transport::iterator::{PreparedIteratorConfig, RowIterator};
-use crate::transport::load_balancing::{
-    LoadBalancingPolicy, RoundRobinPolicy, Statement, TokenAwarePolicy,
-};
+use crate::transport::load_balancing::{DumbPolicy, LoadBalancingPolicy, Statement};
```

Comment:
Ah, ok, it's transitional. Still, each commit deserves a short commit message to introduce the reviewer into your line of thought